ci-operator/config/netobserv/goflow2-kube-enricher/netobserv-goflow2-kube-enricher-main.yaml

```diff
@@ -26,14 +25,10 @@ resources:
       memory: 200Mi
 tests:
 - as: ci
-  commands: |
-    export HOME="/tmp"
-    export GOCACHE=/tmp/.cache
-    export XDG_CACHE_HOME=/tmp/.cache
```

Comment:
I remember some folder permission issues with the make commands without those env vars. Have you verified that they only apply to make test?

Reply:
Actually, I didn't remove these settings but the github `diff` is screwed up here :-/ 
The file on the left is a removed file for operator/4.10 (which I completely removed) whereas the file on the right is a new one for goflow2-kube-enricher